index.html

```diff
@@ -629,6 +629,16 @@ <h2>Profile designator and specifications</h2>
               <td> <a rel="nofollow" class="external text" href="http://www.w3.org/AudioVideo/TT/">TTWG</a>
               </td>
             </tr>
+            <tr>
+              <td> <code>nst1</code>
+              </td>
+              <td> <code>https://www.netflix.com/ns/imsc1.1/text/1</code>
+              </td>
+              <td> <a rel="nofollow" class="external text" href="https://www.netflix.com/ns/imsc1.1/text/1">Netflix IMSC 1.1 Text Profile</a>
```

Comment:
Usually (but oddly, not in the case of IRT, for reasons I can't recall) we add a bibliography entry and reference that instead of a direct href out. @cconcolato please could you add one to the section starting at line 69 and then reference it here with the `[[!ref]]` Respec syntax?

Reply:
yeah, I followed IRT's example. I will update.